test/test_torch.py

```diff
@@ -6180,6 +6180,14 @@ def test_comparison_ops(self):
         for idx in iter_indices(x):
             self.assertEqual(x[idx] >= y[idx], ge[idx] == 1)
 
+    def test_comparison_ops_diff_types(self):
```

Comment:
It would be nice to split this into 3 different tests, each with a more descriptive name. We're testing the following three things. Alternatively, please add comments to each of the below cases describing what is being tested, it's not 100% clear from the error messages what is going on.
1. `test_comparison_ops_must_take_bool_output`
2. `test_inplace_comparison_ops_require_inputs_have_same_dtype`
3. `test_comparison_ops_check_for_scalar_overflow`

Reply:
Fixed